orc8r/gateway/python/magma/magmad/rpc_servicer.py

```diff
@@ -224,6 +228,28 @@ def GenericCommand(self, request, context):
                 grpc.StatusCode.DEADLINE_EXCEEDED,
                 'Command timed out',
             )
+        except CommandExecutionConfigurationException as e:
+            logging.warning(
+                'Error running command %s! %s: %s',
+                request.command, e.__class__.__name__, e,
+            )
+            future.cancel()
+            set_grpc_err(
+                context,
+                grpc.StatusCode.UNIMPLEMENTED,
```

Comment:
I think UNIMPLEMENTED has a special meaning in GRPC and we shouldn't use it here.

Probably, INVALID_ARGUMENT would actually fit well to describe both errors. (And since we map them both to NOT FOUND in Go, we don't really need to distinguish.)

Reply:
Ah okay, I thought it sounded the most reasonable based on https://grpc.github.io/grpc/python/grpc.html#grpc-status-code -> "Operation is not implemented or not supported/enabled in this service." but I'm okay with changing it.

Btw I also updated the Go code so that we don't map them to Not Found: https://github.com/magma/magma/pull/14014/files#diff-10860bcde5a52b147d56694dd62c3421df84de3937d32ca13041775c8541d0c4R136

Reply:
I think it usually means that the GRPC method itself is not implemented on the server. But you're right, from the official description it's not so clear.

The Go code still returns HTTP Not Found though?

Reply:
Right, I see.

Oh yes, that's right, I see what you mean now.